homeassistant/components/micropel/communicator/tcp_client.py

```diff
@@ -0,0 +1,74 @@
+"""TCP client."""
+import logging
+import socket
+import threading
+from typing import Optional
+
+from ..helper.crypto import Crypto
+from ..helper.exceptions import ExceptionResponse, MicropelException
+from ..helper.message import Message
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class TcpClient:
```

Comment:
Communication with external services has to be done with an (PyPi) package -> https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices

Reply:
OK, I agree. I will move it to PyPi package.